manifests/m/Meld/Meld/3.22.0/Meld.Meld.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.4 $debug=AUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Meld.Meld
+PackageVersion: 3.22.0
+PackageLocale: en-US
+Publisher: The Meld project
```

```suggestion
Publisher: Kai Willadsen
```
This must match Control Panel.